tensorflow/lite/micro/python/interpreter/src/tflm_runtime.py

```diff
@@ -37,8 +38,16 @@ def __init__(self,
     if arena_size is None:
       arena_size = len(model_data) * 10
 
+    # Some models make use of resource variables ops, get the count here
+    if num_resource_variables != 0:
+      print(
+          'WARNING: num_resource_variables is deprecated. Resource variable count is ',
+          'calculated automatically.')
+    kNumResourceVariables = flatbuffer_utils.count_resource_variables(
```

Comment:
Please rename this local variable according to our style guide: go/pystyle#naming. The `k` prefix typically indicates constants in C++.

Reply:
Done. Eventually, after we remove the parameter, it will just become 'num_resource_variables'